CI and helm stuff

r7vme opened the pull request:
Add CI and Helm stuff. Plus some changes required for out CI.

xh3b4sd commented on helm/calico-node-controller-chart/templates/configmap.yaml:
```diff
@@ -0,0 +1,10 @@
+  namespace: kube-system
+data:
+  etcd_endpoints: {{ .Values.Installation.V1.Infra.CalicoNodeController.EtcdEndpoint }}
+  etcd_ca: {{ .Values.Installation.V1.Infra.CalicoNodeController.EtcdCaCert }}
```
Nitpicking, the wording I am aware of so far was `crt`, not `cert`. Would like to keep it that way. Also to keep the naming consistent, it should be `etcd_ca` and `EtcdCA`.

xh3b4sd commented on controller.go:
```diff
@@ -24,14 +26,41 @@ import (
+		fmt.Printf("OS / Arch:      %s / %s\n", runtime.GOOS, runtime.GOARCH)
+		fmt.Printf("Source:         %s\n", source)
+		return
+	}
```
Sweet.

xh3b4sd commented on helm/calico-node-controller-chart/templates/serviceaccount.yaml:
```diff
@@ -0,0 +1,5 @@
+kind: ServiceAccount
+metadata:
+  name: calico-node-controller
+  namespace: kube-system
```
For my interest, why is there a service account manifest? I never saw it for our other services.

calvix replied:
RBAC would be my guess

r7vme replied:
exactly, at the moment it's not used. As we do not use RBAC in host cluster.

r7vme commented:
@xh3b4sd addressed your comments regarding naming [here](https://github.com/giantswarm/calico-node-controller/pull/5/commits/39c1b3e763146cd740351a17fc54cb5b59683520). 

Reflected changes also [here](https://github.com/giantswarm/installations/pull/117)
